README.md

```diff
@@ -1,17 +1,39 @@
-# SOEN341-InsertName
+﻿# SOEN341-InsertName
 ### Authors
-* Simon Lim (tech lead) | 
-* Émilie Martin | @ emilie-martin
-* Daniela Venuta | @ daniela-venuta
-* Minh-Tam Do |
-* Luc Nguyen |
-* Karin Kazarian |
+* Simon Lim (tech lead) | @nasaku898
+* Émilie Martin | @emilie-martin
+* Daniela Venuta | @daniela-venuta
+* Minh-Tam Do | @mint47
+* Luc Nguyen | @1999lucnguyen
+* Karin Kazarian | @Karin-kazarian
+
+### Description
+This is a project partial fulfillement of the requirements for the course SOEN341 at Concordia University.
+### Objectives
+The app is a replication of the social media Instagram from which we will add on features.
+### Core Features
```

Comment:
I think it could stand to perhaps give a little rundown of each service? As in what each service is supposed to do (from the Program Breakdown).